util/topgen/templates/plic_all_irqs_test.c.tpl

```diff
@@ -129,6 +133,9 @@ void ottf_external_isr(uint32_t *exc_info) {
     // TODO: Check Interrupt type then clear INTR_TEST if needed.
     CHECK_DIF_OK(dif_csrng_irq_force(&csrng, irq, false));
     CHECK_DIF_OK(dif_csrng_irq_acknowledge(&csrng, irq));
+% else:
+  if (false) {
+% endif
```

Comment:
This is a really clever trick to avoid having to mess around with the formatting below. Nice! :-)